standalone/src/handlers.rs

```diff
@@ -1,34 +1,59 @@
-use crate::types::ProcessBody;
 use crate::types::{AppError, ProcessResponse, Result};
 #[cfg(feature = "neo4j")]
 use ast::lang::graphs::graph_ops::GraphOps;
-use ast::lang::Graph;
 use ast::repo::Repo;
 use axum::Json;
-use lsp::git::{get_commit_hash, git_pull_or_clone};
-use tracing::info;
-pub async fn process(Json(body): Json<ProcessBody>) -> Result<Json<ProcessResponse>> {
-    let repo_url = body.repo_url.clone();
-    let username = body.username.clone();
-    let pat = body.pat.clone();
+pub async fn process() -> Result<Json<ProcessResponse>> {
+    let repo_path = env_not_empty("REPO_PATH");
```

Comment:
I actually like the repo_url and repo_path being in the body instead. Why not support both? they can exist in the env, but be overwritten by values in the body.